src/sdk/sdk.ts

```diff
@@ -33,6 +36,14 @@ export class FusionSDK {
         return this.api.getActiveOrders(request)
     }
 
+    async getOrderStatus({
+        orderHash
+    }: OrderStatusParams): Promise<OrderStatusResponse> {
```

Comment:
Here you can use just `orderHash: string` as a parameter

Reply:
If we decide to change the API, we will need to break the contract 🤔  by refactoring back to the object

Reply:
I don't think this may changed in the future